README.md

```diff
@@ -1,20 +1,151 @@
-* [Kaoto](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-kaoto). It provides a graphical editor for Camel routes.
-* [Java Extension Pack](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack). It provides a large set of tooling for Java applications.
-* [SpringBoot extension pack](https://marketplace.visualstudio.com/items?itemName=Pivotal.vscode-boot-dev-pack) It provides a large set of tooling for SpringBoot applications.
-* [OpenShift Connector](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-openshift-connector). It provides a large set of tooling to interact with an OpenShift instance.
-* [XML Language Support](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-xml). It provides tooling for XML files.
-* [AtlasMap Data Transformation editor](https://marketplace.visualstudio.com/items?itemName=redhat.atlasmap-viewer). It provides a graphical editor for Data Transformation based on the camel-atlasmap component.
-* [Tools for Microprofile](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-microprofile). It provides support for the development of MicroProfile®-based applications.
-* [Quarkus tools for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus). It provides Quarkus and MicroProfile development by extending Visual Studio Code extension for MicroProfile with Quarkus features.
-* *Deprecated* [Project initializer by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.project-initializer). It allows to create Apache Camel or Red Hat Fuse projects by selecting a mission.
-![Create project](./images/CreateProjectFromTopLevelCommand.gif)
+<h1 align="center">
+  <img width="125" height="125" src="https://raw.githubusercontent.com/camel-tooling/vscode-camel-extension-pack/main/icons/icon128.png">
+  <br>
+  Extensions Pack for Apache Camel
+</h1>
+
+<p align="center">
+  <a href="https://marketplace.visualstudio.com/items?itemName=redhat.apache-camel-extension-pack"><img src="https://img.shields.io/visual-studio-marketplace/v/redhat.apache-camel-extension-pack?style=for-the-badge" alt="Marketplace Version"/></a>
+  <a href="https://github.com/camel-tooling/vscode-camel-extension-pack/actions/workflows/main.yml"><img src="https://img.shields.io/github/actions/workflow/status/camel-tooling/vscode-camel-extension-pack/main.yml?style=for-the-badge" alt="Main CI"></a>
+  <a href="https://github.com/camel-tooling/vscode-camel-extension-pack/blob/main/LICENSE"><img src="https://img.shields.io/github/license/camel-tooling/vscode-camel-extension-pack?color=blue&style=for-the-badge" alt="License"/></a>
+  <a href="https://camel.zulipchat.com/#narrow/stream/258729-camel-tooling"><img src="https://img.shields.io/badge/zulip-join_chat-brightgreen?color=yellow&style=for-the-badge" alt="Zulip"/></a>
+</p><br/>
+
+<h2 align="center">Camel Extensions Pack for Visual Studio Code</h2>
```

Comment:
for legal reasons we cannot mention it this way. Given that we are not an official Apache project, it must be "Something for Camel", and never "Camel XXX"

Reply:
I will probably remove this `h2` heading because I do not have idea what to mention here to not duplicate h1